### ISSUE TITLE
[Bug] ref_kl is set to True by default in method/alerdistill.yaml

### ISSUE THREAD

**Issue by yanwj21** [dakfjalka/aler-distill#1]
Dear Author,

Thanks for your great work!

I recently applied this method to our CPT (Continuous Pre-training) data but observed that the performance on the target test set was significantly worse than the SFT baseline.

After investigation, I found that ref_kl loss is set to True by default in method/alerdistill.yaml. However, according to the original paper, this loss is not a requirement for the method. When I manually set ref_kl to False, the performance returned to expected levels (comparable to SFT).

It seems this might have been a temporary setting for an SFT+KL ablation study? If so, I suggest changing the default value to False to align with the paper and ensure better out-of-the-box performance.

Best regards.

**Comment by dakfjalka** [comment]
PR merged. Thanks a lot!
